snapshotter/BUILD.gn

```diff
@@ -1,30 +0,0 @@
-# Copyright 2014 The Chromium Authors. All rights reserved.
-# Use of this source code is governed by a BSD-style license that can be
-# found in the LICENSE file.
-
-executable("snapshotter") {
-  output_name = "sky_snapshot"
-
-  sources = [
-    "main.cc",
-  ]
-
-  deps = [
-    "//dart/runtime:libdart",
-    "//dart/runtime/vm:libdart_platform",
-    "//flutter/lib/snapshot",
-    "//lib/ftl",
-    "//lib/tonic/converter",
-    "//lib/tonic/file_loader",
-  ]
-
-  if (!is_ios && !is_mac && !is_fuchsia) {
-    ldflags = [
-      # libgcc_s is not a DT_NEEDED library; it normally gets loaded implicitly.
-      # This flag ensures that rpath is referenced when searching for
-      # the so, which in turn gets the resulting sky_snapshot binary
-      # closer to being able to run in Google's production environment.
-      "-lgcc_s",
```

Comment:
We might need this craziness for gen_snapshot when the time comes to integrate with Google's production environment.